src/connectors/rating-menu/connectRatingMenu.js

```diff
@@ -180,9 +185,7 @@ export default function connectRatingMenu(renderFn, unmountFn = noop) {
       dispose({ state }) {
         unmountFn();
 
-        const nextState = state
-          .removeDisjunctiveFacetRefinement(attribute)
-          .removeDisjunctiveFacet(attribute);
+        const nextState = state.removeDisjunctiveFacet(attribute);
```

Comment:
looks like there's no tests yet for `dispose`

This could also be directly returned

Reply:
WDYM? I created a test suite.

Reply:
I missed it in the diff, sorry